### PR TITLE
[UI] Use cover art for the most recently played game

### PR DESCRIPTION
**Pull request by redromnon**
Although this is not part of the Figma design, I thought of using the cover art for the just recently played game. Kind of like how several game launchers do.

This PR introduces the `justPlayed` property for the `Card` interface that is set to`true` for the first game under the Recently Played `GamesList`. If `true`, the `GameCard`' uses game's `art_cover` as the `src`.

The covert art box occupies two grids for correctly containing the art.

Screenshot from 2023-05-31 08-05-06

---

Use the following Checklist if you have changed something on the Backend or Frontend:

- [ ] Tested the feature and it's working on a current and clean install.
- [ ] Tested the main App features and they are still working on a current and clean install. (Login, Install, Play, Uninstall, Move games, etc.)
- [ ] Created / Updated Tests (If necessary)
- [ ] Created / Updated documentation (If necessary)

### CODE REVIEW

File: src/frontend/screens/Library/components/GamesList/index.tsx [hunk 1]
```diff
@@ -90,9 +90,12 @@ const GamesList = ({
...
       )}
       {!!library.length &&
-        library.map((gameInfo) => {
+        library.map((gameInfo, index) => {
           const { app_name, is_installed, runner } = gameInfo
-
+          let isJustPlayed = false
+          if (isRecent && index === 0) {
+            isJustPlayed = true
+          }
```
Comment (arielj) [inline src/frontend/screens/Library/components/GamesList/index.tsx:98]:
```suggestion
          const isJustPlayed = isRecent && index === 0
```

File: src/frontend/screens/Library/index.css [hunk 2]
```diff
@@ -11,6 +11,11 @@
   padding: 0 var(--space-md-fixed) var(--space-md-fixed);
 }
 
+.gameList.firstLane > :first-child {
+  grid-column-start: 1;
+  grid-column-end: 3;
+}
```
Comment (arielj) [inline src/frontend/screens/Library/index.css:17]:
```suggestion
.justPlayed {
  grid-column: span 2;
}
```

File: src/frontend/screens/Library/components/GameCard/index.css [hunk 3]
```diff
@@ -106,6 +120,12 @@
   padding: var(--space-xs-fixed);
 }
 
+.gameCard.justPlayed > .icons {
+  justify-content: space-between;
+  grid-template-columns: 0.15fr 0.15fr 0.15fr;
```
Comment (arielj) [inline src/frontend/screens/Library/components/GameCard/index.css:125]:
```suggestion
  grid-template-columns: 1fr 1fr 1fr;
```

Comment (redromnon) [reply src/frontend/screens/Library/components/GameCard/index.css:125]: Screenshot from 2023-06-06 15-27-03

The suggested change will end up looking like this though. Do you prefer this?

Comment (redromnon) [reply src/frontend/screens/Library/components/GameCard/index.css:125]: Or we could discuss the ideal layout in Discord with the others' feedback.

**Comment by redromnon** [comment]
> For me, something like this is more consistent:

I had tried placed these buttons in that alignment too. Yes, it looks much better.

**Comment by redromnon** [comment]
> I think it is too much space on the button. For me, something like this is more consistent: telegram-cloud-photo-size-1-4915798621455232963-y
> 
> But maybe @biliesilva can give more insights on that.

I've made the logo alignment look like the above screenshot.

**Comment by flavioislima** [comment]
@redromnon can you make a version of it where you replace the play icons with a button with the text PLAY on it? Since we have more space now.

**Comment by redromnon** [comment]
Just replace the green play button with text while keeping the settings and update icons?

**Comment by redromnon** [comment]
Here's how the new changes look:
Screenshot from 2023-07-03 19-48-41

Also decided to move the update icon towards the opposite end. I feel it makes it look the alignment look better.

**Review by flavioislima — APPROVED** [review]
Looks good 👍🏽

**Pull request merged**